controllers/profiles/common/app_properties.go

```diff
@@ -50,6 +51,7 @@ var immutableApplicationProperties = "quarkus.http.port=" + DefaultHTTPWorkflowP
 var _ AppPropertyHandler = &appPropertyHandler{}
 
 type AppPropertyHandler interface {
+	asService() AppPropertyHandler
```

Comment:
You don't need to add a private method to the interface.

Reply:
good catch, i had originally made it public

Reply:
i've decided to do away with that method entirely... seems cleaner.